js/client.js

```diff
@@ -931,7 +953,8 @@
 					name: name,
 					userid: userid,
 					named: named,
-					avatar: parts[3]
+					avatar: parts[3],
+					settings: JSON.parse(parts[4]),
```

Comment:
A trailing comma? I'm surprised ESLint isn't yelling at you. This is the client! Safari 5 doesn't support trailing commas.